docs/UserGuide.md

```diff
@@ -667,69 +667,74 @@ _Add screenshot here_
 
 <br>
 
-### Listing all patients for today: `view` `--today`
+### Listing all tasks for today: `view` `--today`
 
-Shows a list of all patients with tasks due today.
+You can view the list of tasks that are due today using the `view` command with the special flag `--today`.
 
 Format: `view --today`
```

Comment:
Would be good to bold the command word for consistency. (most of the other features have it)
```suggestion
Format: **`view`**`--today`
```